core/src/test/java/cocktailofferer/service/image/query/ImageMetadataQueryServiceTestConfiguration.java

```diff
@@ -0,0 +1,9 @@
+package cocktailofferer.service.image.query;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.ComponentScan;
+
+@ComponentScan
+@TestConfiguration
+public class ImageMetadataQueryServiceTestConfiguration {
```

Comment:
Having a config inside test class is less error-prone

Reply:
Why do u think so?